analyses/create-subset-files/create_subset_files.sh

```diff
@@ -31,6 +31,8 @@ cd "$script_directory" || exit
 # generated via these scripts
 FULL_DIRECTORY=../../data/$RELEASE
 SUBSET_DIRECTORY=../../data/testing/$RELEASE
+# If run subsetting only, we need to make this directory
+mkdir -p $SUBSET_DIRECTORY
```

Comment:
This actually needed brackets because it was still throwing an error, so I updated in my branch and added logic to make if does not exist.

${SUBSET_DIRECTORY}